sdk/search/search-documents/review/search-documents.api.md

```diff
@@ -986,11 +986,58 @@ export enum KnownEntityRecognitionSkillLanguage {
 
 // @public
 export enum KnownImageAnalysisSkillLanguage {
+    Ar = "ar",
```

Comment:
Is the way we're "Enabling" the skills just us exposing the full list of supported languages for them?

Reply:
Yes, I think it's just exposing the model to clients. The [.NET implementation](https://github.com/Azure/azure-sdk-for-net/pull/31176/files) suggests the same.